standalone/react-tooltip.js

```diff
@@ -536,6 +544,7 @@ var ReactTooltip = (0, _staticMethods2.default)(_class = (0, _windowListener2.de
       });
 
       if (globalEventOff) window.removeEventListener(globalEventOff, this.hideTooltip);
+      window.removeEventListener('click', this.onClickOutsideFrozenTooltip);
```

Comment:
Hmmm... I don’t know why GitHub says this line was added, because it wasn’t! You hadn’t forgotten about it. ;)